sirepo/package_data/static/js/sirepo.js

```diff
@@ -1556,14 +1544,155 @@ SIREPO.app.service('validationService', function(utilities) {
 
 });
 
+SIREPO.app.factory('srCache', function(appState, $rootScope) {
+
```

Comment:
@e-carlin and @robnagler srCache is the client side cache for web sockets, using indexedDB